src/components/subscriptions/TabContentTable.jsx

```diff
@@ -106,36 +114,53 @@ export default function TabContentTable() {
           />
         )}
       </div>
-      {tableData.length > 0 ? (
-        <React.Fragment>
-          <div className="table-responsive">
-            <Table
-              data={tableData}
-              columns={columns}
-              className="table-striped"
-            />
-          </div>
-          <div className="mt-3 d-flex justify-content-center">
-            <Pagination
-              // eslint-disable-next-line no-console
-              onPageSelect={page => console.log(page)}
-              pageCount={Math.ceil(users.count / PAGE_SIZE)}
-              currentPage={1}
-              paginationLabel={activeTabData.paginationLabel}
-            />
-          </div>
-        </React.Fragment>
-      ) : (
-        <React.Fragment>
-          <hr className="mt-0" />
+      {isLoading && <LoadingMessage className="loading mt-3" />}
+      {
+        errors && Object.entries(errors).map(([title, message]) => (
           <StatusAlert
-            alertType="warning"
-            dialog={activeTabData.noResultsLabel}
-            dismissible={false}
-            open
+            className="mt-3"
+            alertType="danger"
+            iconClassName="fa fa-times-circle"
+            title={`Unable to load data for ${title}`}
+            message={`Try refreshing your screen (${message})`}
+            key={title}
           />
+         ))
+      }
+      { !errors &&
```

Comment:
+1 on hiding the table

Reply:
Yes, I wasn't sure about which behavior would be most appropriate. Previously, I was conditionally rendering table on `isLoading` flag, but in this case, there was flakiness for date re-fetching (as `isLoading` is true while re-fetching subscription data). I thought it would be a better experience for the user to keep the table as is instead of hiding it momentarily and show it again while re-fetching the data.

Reply:
By having the loading message appear above the table, it would make the entire table move around any time its loading, which could get annoying. I think a slightly more subtle approach would be to hide the table when loading.

That said, I agree it's not ideal in either case. I think hiding the table and showing the loading message (with a reasonable height, like 360px) is fine for now, though, and we can revisit this UX later.